hs_core/templates/pages/genericresource.html

```diff
@@ -208,12 +210,14 @@ <h4 class="modal-title">Confirm file deletion</h4>
                             {#  Resource specific metadata can't be edited for HIS Referenced Time Series #}
                             {% if cm|resource_type != "HIS Referenced Time Series" or not metadata_form %}
                                 {% if  cm|resource_type != "Generic" %}
-                                    <li class="col-xs-2 col-md-2">
-                                        <a href="#resourceSpecificTab" data-toggle="tab" title="Resource specific">
-                                            <span class="glyphicon glyphicon-folder-close"></span>
-                                            &nbspResource Specific
-                                        </a>
-                                    </li>
+                                    {% if  cm|resource_type|lower != "collection resource" %}
```

Comment:
@zhiyuli Generic resources should not have a resource specific tab either, so you should retain the check for Generic resources rather than replace it with collection resources. Also, collection resource type in lowercase should be "collectionresource" without space in between, right? If so, you if check logic is not correct.